make/ModuleWrapper.gmk

```diff
@@ -45,6 +45,18 @@ TARGETS :=
 # Include the file being wrapped.
 include $(MAKEFILE_PREFIX).gmk
 
+ifeq ($(MAKEFILE_PREFIX), Lib)
+  ifneq ($($(MODULE)_JDK_LIBS), )
+    LIBLIST := $(SUPPORT_OUTPUTDIR)/modules_static-libs/$(MODULE)/module-libs.txt
+
+    $(LIBLIST): $(TARGETS)
+	$(call MakeDir, $(@D))
+	$(ECHO) $($(MODULE)_JDK_LIBS) > $@
+
+    TARGETS += $(LIBLIST)
+  endif
+endif
+
```

Comment:
This part looks a bit convoluted. It would be nice with a comment explaining what it does, where `$($(MODULE)_JDK_LIBS)` is coming from and what consumes the module-libs.txt.

Reply:
Fixed.